src/cairo_types/syscalls.rs

```diff
@@ -43,3 +44,26 @@ pub struct NewStorageWriteRequest {
     pub key: Felt252,
     pub value: Felt252,
 }
+
+#[derive(FieldOffsetGetters)]
+pub struct TxInfo {
+    /// The version of the transaction. It is fixed (currently, 1) in the OS, and should be
+    /// signed by the account contract.
+    /// This field allows invalidating old transactions, whenever the meaning of the other
+    /// transaction fields is changed (in the OS).
+    pub version: Felt252,
+    /// The account contract from which this transaction originates.
+    pub account_contract_address: Felt252,
+    /// The max_fee field of the transaction.
+    pub max_fee: Felt252,
+    /// The signature of the transaction.
+    pub signature_len: Felt252,
+    pub signature: Relocatable,
```

Comment:
Why not `Felt252`? Other pointer are modeled with Felts.

Reply:
Some structures use `Relocatable` for `felt*`. I'm keeping this information just in case I want to update the API of the `CairoType`/`FieldOffsetGetters` macros and add methods.